core/terraform/scripts/letsencrypt.sh

```diff
@@ -92,6 +92,11 @@ ledir=$(pwd)/letsencrypt
 
 mkdir -p "${ledir}/logs"
 
+CERT_FQDN=$FQDN
+if [[ -n "$CUSTOM_DOMAIN" ]]; then
+  CERT_FQDN=$CUSTOM_DOMAIN
+fi
+
```

Comment:
could you add that log line about the domain being used?

Reply:
Added